lib/builders/latexmk.js

```diff
@@ -4,6 +4,8 @@ import childProcess from 'child_process'
 import path from 'path'
 import Builder from '../builder'
 
+const latexRegExp = /^latex|u?platex$/
```

Comment:
I want to switch to uppercasing these module scoped constants. This should be renamed to `LATEX_PATTERN`.